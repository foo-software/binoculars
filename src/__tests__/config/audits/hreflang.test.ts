import auditHreflang from '../../../config/audits/hreflang';

describe('hreflang audit', () => {
  it('should match snapshot', () => {
    expect(auditHreflang('en')).toMatchSnapshot();
  });
});
