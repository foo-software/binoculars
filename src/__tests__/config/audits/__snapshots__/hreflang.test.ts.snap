// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hreflang audit should match snapshot 1`] = `
Object {
  "description": "hreflang links tell search engines what version of a page they should list in search results for a given language or region. [Learn more](https://web.dev/hreflang/).",
  "title": "Document has a valid \`hreflang\`",
}
`;
